kubernetes/coturn.yml.erb

```diff
@@ -0,0 +1,104 @@
+---
+apiVersion: v1
+data:
+  # see https://github.com/matrix-org/synapse/blob/master/docs/turn-howto.md#configuration
+  turnserver.conf: |
```

Comment:
imo we should build this into the container, like I mentioned in #11

Reply:
Hmm... it's not fully static because it needs the `INTERNAL_IP` variable. I'll need to look into why it needs that in the first place.